[Bug]: Cannot load service: liquibase.hub.HubService

### What happened?

When starting Airsonic I see this error in the logs:
2023-06-05 10:30:02.670  INFO --- l.s.StandardServiceLocator               : Cannot load service: liquibase.hub.HubService: Provider liquibase.hub.core.StandardHubService could not be instantiated

### Steps to reproduce

Start Airsonic

### Version

11.1.x (Edge)

### Version Detail

_No response_

### Operating System

Synology Docker

### Java Version

Apache Tomcat/9.0.74, java 17.0.5, Linux (1,07 GB / 1,67 GB)

### Database

Other

### DB Detail

_No response_

### Configuration paramter

# Airsonic preferences.  NOTE: This file is automatically generated. Do not modify while application is running

# Airsonic preferences.  NOTE: This file is automatically generated. Do not modify while application is running
EncryptionKeyPassword=removed
EncryptionKeySalt=removed
SettingsChanged=1685093141376
JWTKey=9eirnhrnn3pg65e1j0l6bjvteq
DlnaEnabled=false
DlnaServerName=Airsonic
DlnaBaseLANURL=http://192.168.178.25/airsonic/
DlnaServerId=8c88e797-a32e-4380-8cb1-537d025239de
IndexString=A B C D E F G H I J K L M N O P Q R S T U V W X-Z(XYZ)
IgnoredArticles=The El La Los Las Le Les De Het Een
GenreSeparators=;
Shortcuts=
PlaylistFolder=/playlists
MusicFileTypes=mp3 ogg oga aac m4a m4b flac wav wma aif aiff ape mpc shn mka opus alm 669 mdl far xm mod fnk imf it liq wow mtm ptm rtm stm s3m ult dmf dbm med okt emod sfx m15 mtn amf gdm stx gmc psm j2b umx amd rad hsc flx gtk mgt mtp wv
VideoFileTypes=flv avi mpg mpeg mp4 m4v mkv mov wmv ogv divx m2ts webm
CoverArtFileTypes2=cover.jpg cover.png cover.gif folder.jpg jpg jpeg gif png
CoverArtSource=FILETAG
CoverArtConcurrency=4
CoverArtQuality=90
SortAlbumsByYear=true
GettingStartedEnabled=false
WelcomeTitle=Sepulchrijn: Bart's Muziek - Kaji
WelcomeSubtitle=
WelcomeMessage2=
LoginMessage=
server.servlet.session.timeout=3600m
Theme=ripserver
LocaleLanguage=nl
LocaleCountry=
LocaleVariant=
IndexCreationInterval=1
IndexCreationHour=2
FastCacheEnabled=false
OrganizeByFolderStructure=true
UploadsFolder=%{['USER_MUSIC_FOLDERS'][0]}/Incoming
ExcludePattern=#recycle|@eaDir
IgnoreSymLinks=true
DownloadBitrateLimit=0
UploadBitrateLimit=0
LdapEnabled=false
LdapUrl=ldap://host.domain.com:389/cn=Users,dc=domain,dc=com
LdapSearchFilter=(sAMAccountName={0})
LdapManagerDn=
LdapAutoShadowing=false
SmtpServer=mail.gufp.nl
SmtpEncryption=SSL/TLS
SmtpPort=465
SmtpUser=bart@gufp.nl
SmtpFrom=airsonic@gufp.nl
SmtpPassword=55746b576869303031302121
CaptchaEnabled=false
ReCaptchaSiteKey=removed
HideIndexedFiles=false
PodcastUpdateInterval=-1
PodcastEpisodeRetentionCount=2
PodcastEpisodeDownloadCount=0
spring.liquibase.parameters.mysqlVarcharLimit=384
DbBackupUpdateInterval=-1
DbBackupRetentionCount=5
EnableCueIndexing=true

### Proxy Server

_No response_

### client detail

All

### language

Dutch

### Relevant log output

```console
Docker USER id: 0
Docker PUID env: 1026
Docker USER group: 0
Docker PGID env: 100
Process will run as:
User: 1026
Group: 100
openjdk 17.0.5 2022-10-18
OpenJDK Runtime Environment Temurin-17.0.5+8 (build 17.0.5+8)
OpenJDK 64-Bit Server VM Temurin-17.0.5+8 (build 17.0.5+8, mixed mode, sharing)
JAVA_OPTS=-Xmx4096M -Dserver.forward-headers-strategy=NATIVE
ffmpeg version 4.4.2-0ubuntu0.22.04.1 Copyright (c) 2000-2021 the FFmpeg developers
built with gcc 11 (Ubuntu 11.2.0-19ubuntu1)
configuration: --prefix=/usr --extra-version=0ubuntu0.22.04.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-pocketsphinx --enable-librsvg --enable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
libavutil      56. 70.100 / 56. 70.100
libavcodec     58.134.100 / 58.134.100
libavformat    58. 76.100 / 58. 76.100
libavdevice    58. 13.100 / 58. 13.100
libavfilter     7.110.100 /  7.110.100
libswscale      5.  9.100 /  5.  9.100
libswresample   3.  9.100 /  3.  9.100
libpostproc    55.  9.100 / 55.  9.100
curl 7.81.0 (x86_64-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.13
Release-Date: 2022-01-05
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets zstd
PATH=/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
CONTEXT_PATH=/
           _                       _          
     /\   (_)                     (_)         
    /  \   _ _ __  ___  ___  _ __  _  ___     
   / /\ \ | | '__|/ __|/ _ \| '_ \| |/ __|    
  / ____ \| | |   \__ \ (_) | | | | | (__     
 /_/    \_\_|_|   |___/\___/|_| |_|_|\___|    
                                               
                        
2023-06-05 10:29:31.738  INFO --- org.airsonic.player.Application          : Starting Application using Java 17.0.5 on 00cd43cedecd with PID 1 (/app/WEB-INF/classes started by au in /var)
2023-06-05 10:29:31.744  INFO --- org.airsonic.player.Application          : No active profile set, falling back to 1 default profile: "default"
2023-06-05 10:29:56.885  INFO --- l.database.core.HsqlDatabase             : Set default schema name to PUBLIC
2023-06-05 10:29:57.358  INFO --- l.l.StandardLockService                  : Successfully acquired change log lock
2023-06-05 10:30:01.666  INFO --- l.c.StandardChangeLogHistoryService      : Reading from PUBLIC.DATABASECHANGELOG
2023-06-05 10:30:02.074  INFO --- l.l.StandardLockService                  : Successfully released change log lock
2023-06-05 10:30:02.325  INFO --- l.l.StandardLockService                  : Successfully acquired change log lock
2023-06-05 10:30:02.390  INFO --- l.c.StandardChangeLogHistoryService      : Reading from PUBLIC.DATABASECHANGELOG
2023-06-05 10:30:02.670  INFO --- l.s.StandardServiceLocator               : Cannot load service: liquibase.hub.HubService: Provider liquibase.hub.core.StandardHubService could not be instantiated
2023-06-05 10:30:02.843  INFO --- l.l.StandardLockService                  : Successfully released change log lock
2023-06-05 10:30:03.531  INFO --- o.a.p.service.SettingsService            : Java: 17.0.5+8, OS: Linux, Memory (max bytes): 4294967296
2023-06-05 10:30:04.394  INFO --- org.airsonic.player.Application          : Detected Tomcat web server
2023-06-05 10:30:21.291  INFO --- o.a.p.s.TaskSchedulingService            : Task remote-playstatus-cleanup scheduled
2023-06-05 10:30:21.739  INFO --- o.airsonic.player.dao.PlayerDao          : Deleted 1 player(s) that haven't been used after 2023-04-06T08:30:21.645478180Z
2023-06-05 10:30:22.706  INFO --- o.a.p.service.PodcastService             : Automatic default Podcast update disabled for podcasts
2023-06-05 10:30:23.666  INFO --- o.a.p.service.DatabaseService            : Automatic DB backup disabled
2023-06-05 10:30:23.896  INFO --- o.a.p.s.search.IndexManager              : Index was found (index version 19). 
2023-06-05 10:30:23.899  INFO --- o.a.p.s.TaskSchedulingService            : Task mediascanner-IndexingTask scheduled
2023-06-05 10:30:23.899  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at 2023-06-06T02:00:00.897302304
2023-06-05 10:30:31.550  INFO --- org.airsonic.player.Application          : Started Application in 65.977 seconds (JVM running for 73.59)
2023-06-05 10:30:47.409  INFO --- o.a.p.f.BootstrapVerificationFilter      : Servlet container: Apache Tomcat/9.0.74
2023-06-05 10:30:47.904  INFO --- o.a.p.s.LoggingExceptionResolver         : 192.168.178.2: Client unexpectedly closed connection while loading https://airsonic.gufp.nl/script/mediaelement/mediaelementplayer.min.css (java.io.IOException: Broken pipe)
2023-06-05 10:30:51.579  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: bart | EventType: CREATED | Old value: null | New value: bart ["SETTINGS","STREAM","UPLOAD","PODCAST","ADMIN","DOWNLOAD","COMMENT","COVERART","JUKEBOX","SHARE"]
2023-06-05 10:30:52.078  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: bart | EventType: CREATED | Old value: null | New value: org.airsonic.player.domain.UserSettings@47ff0fb7
2023-06-05 10:30:52.825  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: -0--1.0 | EventType: CREATED | Old value: null | New value: Music
2023-06-05 10:30:52.836  INFO --- o.a.p.service.VersionService             : Resolved local Airsonic version to: 11.1.3-SNAPSHOT.20230527120152
2023-06-05 10:30:53.968  INFO --- o.a.p.s.LoggingExceptionResolver         : 192.168.178.2: Client unexpectedly closed connection while loading https://airsonic.gufp.nl/icons/ripserver/up.gif (java.io.IOException: Broken pipe)
2023-06-05 10:30:53.969  INFO --- o.a.p.s.LoggingExceptionResolver         : 192.168.178.2: Client unexpectedly closed connection while loading https://airsonic.gufp.nl/icons/ripserver/play.gif (java.io.IOException: Broken pipe)
2023-06-05 10:30:56.647  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: 101394 | EventType: CREATED | Old value: null | New value: Day Dreams
2023-06-05 10:30:56.658  WARN --- o.airsonic.player.util.FileUtil          : Could not get file modify date for /music/B/Big Band/Ed Palermo Big Band Lousy Day in Harlem/01. Laurie Frink.mp3
java.nio.file.NoSuchFileException: /music/B/Big Band/Ed Palermo Big Band Lousy Day in Harlem/01. Laurie Frink.mp3
	at java.base/sun.nio.fs.UnixException.translateToIOException(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(Unknown Source) ~[na:na]
	at java.base/java.nio.file.Files.readAttributes(Unknown Source) ~[na:na]
	at java.base/java.nio.file.Files.getLastModifiedTime(Unknown Source) ~[na:na]
	at org.airsonic.player.util.FileUtil.lastModified(FileUtil.java:54) ~[classes/:na]
	at org.airsonic.player.service.MediaFileService.needsUpdate(MediaFileService.java:198) ~[classes/:na]
	at org.airsonic.player.service.MediaFileService.checkLastModified(MediaFileService.java:204) ~[classes/:na]
	at org.airsonic.player.service.MediaFileService.getMediaFile_aroundBody2(MediaFileService.java:175) ~[classes/:na]
	at org.airsonic.player.service.MediaFileService$AjcClosure3.run(MediaFileService.java:1) ~[classes/:na]
	at org.springframework.cache.aspectj.AbstractCacheAspect.ajc$around$org_springframework_cache_aspectj_AbstractCacheAspect$1$2bc714b5proceed(AbstractCacheAspect.aj:64) ~[spring-aspects-5.3.27.jar:5.3.27]
	at org.springframework.cache.aspectj.AbstractCacheAspect$AbstractCacheAspect$1.invoke(AbstractCacheAspect.aj:71) ~[spring-aspects-5.3.27.jar:5.3.27]
	at org.springframework.cache.interceptor.CacheAspectSupport.invokeOperation(CacheAspectSupport.java:366) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:421) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:345) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.cache.aspectj.AbstractCacheAspect.ajc$around$org_springframework_cache_aspectj_AbstractCacheAspect$1$2bc714b5(AbstractCacheAspect.aj:80) ~[spring-aspects-5.3.27.jar:5.3.27]
	at org.airsonic.player.service.MediaFileService.getMediaFile(MediaFileService.java:168) ~[classes/:na]
	at org.airsonic.player.ajax.BookmarksWSController.bookmarkToBookmarkInfo(BookmarksWSController.java:39) ~[classes/:na]
	at org.airsonic.player.ajax.BookmarksWSController.lambda$0(BookmarksWSController.java:30) ~[classes/:na]
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) ~[na:na]
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[na:na]
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source) ~[na:na]
	at org.airsonic.player.ajax.BookmarksWSController.getBookmarks(BookmarksWSController.java:32) ~[classes/:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:169) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:119) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.handler.invocation.AbstractMethodMessageHandler.handleMatch(AbstractMethodMessageHandler.java:569) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.simp.annotation.support.SimpAnnotationMethodMessageHandler.handleMatch(SimpAnnotationMethodMessageHandler.java:511) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.simp.annotation.support.SimpAnnotationMethodMessageHandler.handleMatch(SimpAnnotationMethodMessageHandler.java:94) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.handler.invocation.AbstractMethodMessageHandler.handleMessageInternal(AbstractMethodMessageHandler.java:524) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.handler.invocation.AbstractMethodMessageHandler.handleMessage(AbstractMethodMessageHandler.java:458) ~[spring-messaging-5.3.27.jar:5.3.27]
	at org.springframework.messaging.support.ExecutorSubscribableChannel$SendTask.run(ExecutorSubscribableChannel.java:144) ~[spring-messaging-5.3.27.jar:5.3.27]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[na:na]
	at java.base/java.lang.Thread.run(Unknown Source) ~[na:na]org.airsonic.player.domain.CoverArt@66024ef6 | New value: null
2023-06-05 10:33:07.659  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: ARTIST-5678 | EventType: REMOVED | Old value: org.airsonic.player.domain.CoverArt@66025352 | New value: null
2023-06-05 10:33:07.659  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: ARTIST-5712 | EventType: CREATED | Old value: null | New value: org.airsonic.player.domain.CoverArt@66025770
2023-06-05 10:33:07.659  INFO --- o.a.p.s.MediaScannerService              : Marking non-present artists.
2023-06-05 10:33:07.659  INFO --- o.a.p.s.CacheConfiguration$CacheLogger   : Cache Key: ARTIST-5712 | EventType: REMOVED | Old value: org.airsonic.player.domain.CoverArt@66025770 | New value: null
2023-06-05 10:33:07.662  INFO --- o.a.p.s.MediaScannerService              : Artist persistence complete
2023-06-05 10:33:15.294  INFO --- o.a.p.s.MediaScannerService              : Marking non-present files.
2023-06-05 10:33:15.329  INFO --- o.a.p.s.MediaScannerService              : File marking complete
2023-06-05 10:33:15.329  INFO --- o.a.p.s.MediaScannerService              : Completed media library scan.
2023-06-05 10:33:28.366  INFO --- o.a.p.s.MediaScannerService              : Media library scan took 131s
2023-06-05 10:33:28.366  INFO --- o.a.p.service.PlaylistService            : Starting playlist import.
2023-06-05 10:33:28.369  INFO --- o.a.p.service.PlaylistService            : Completed playlist import.
```

I have investigated the detailed logs, and it seems that the error is related to the constructor of SnakeYAML. The stack trace indicates that there is an issue with the instantiation of SnakeYAML's constructor, which is causing the error.

```
java.util.ServiceConfigurationError: liquibase.hub.HubService: Provider liquibase.hub.core.StandardHubService could not be instantiated
        at java.base/java.util.ServiceLoader.fail(Unknown Source) ~[na:na]
        at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(Unknown Source) ~[na:na]
        at java.base/java.util.ServiceLoader$ProviderImpl.get(Unknown Source) ~[na:na]
        at java.base/java.util.ServiceLoader$3.next(Unknown Source) ~[na:na]
        at liquibase.servicelocator.StandardServiceLocator.findInstances(StandardServiceLocator.java:24) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.plugin.AbstractPluginFactory.findAllInstances(AbstractPluginFactory.java:92) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.plugin.AbstractPluginFactory.getPlugin(AbstractPluginFactory.java:62) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.hub.HubServiceFactory.getService(HubServiceFactory.java:30) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.hub.HubUpdater.<init>(HubUpdater.java:50) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Liquibase.lambda$update$1(Liquibase.java:260) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Scope.lambda$child$0(Scope.java:180) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Scope.child(Scope.java:189) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Scope.child(Scope.java:179) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Scope.child(Scope.java:158) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Liquibase.runInScope(Liquibase.java:2445) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Liquibase.update(Liquibase.java:235) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.Liquibase.update(Liquibase.java:220) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:328) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:283) ~[liquibase-core-4.18.0.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:657) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:640) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:119) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:399) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1431) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:657) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:640) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:119) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:399) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1431) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:760) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:720) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:119) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:399) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1431) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:213) ~[spring-beans-5.3.27.jar:5.3.27]
        at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1162) ~[spring-context-5.3.27.jar:5.3.27]
        at org.springframework.ui.context.support.UiApplicationContextUtils.initThemeSource(UiApplicationContextUtils.java:58) ~[spring-context-5.3.27.jar:5.3.27]
        at org.springframework.web.context.support.GenericWebApplicationContext.onRefresh(GenericWebApplicationContext.java:199) ~[spring-web-5.3.27.jar:5.3.27]
        at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:160) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:577) ~[spring-context-5.3.27.jar:5.3.27]
        at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:147) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:731) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:408) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:307) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:164) ~[spring-boot-2.7.11.jar:2.7.11]
        at org.airsonic.player.Application.main(Application.java:70) ~[classes/:na]
Caused by: java.lang.NoSuchMethodError: org.yaml.snakeyaml.constructor.Constructor: method 'void <init>()' not found
        at liquibase.hub.core.HttpClient.<init>(HttpClient.java:48) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.hub.core.StandardHubService.createHttpClient(StandardHubService.java:41) ~[liquibase-core-4.18.0.jar:na]
        at liquibase.hub.core.StandardHubService.<init>(StandardHubService.java:37) ~[liquibase-core-4.18.0.jar:na]
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:na]
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[na:na]
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[na:na]
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[na:na]
        at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source) ~[na:na]
        ... 81 common frames omitted

```

--- a/airsonic-main/pom.xml
+++ b/airsonic-main/pom.xml
@@ -16,7 +16,7 @@
         <chameleon.version>2.0.0-RELEASE</chameleon.version>
         <tomcat.server.scope>provided</tomcat.server.scope>
         <!-- For fixing import/export due to https://github.com/liquibase/liquibase/issues/1598 -->
-        <liquibase.version>4.18.0</liquibase.version>
+        <liquibase.version>4.22.0</liquibase.version>
     </properties>
 
     <dependencies>
@@ -451,7 +451,7 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>4.18.0</version>
+            <version>${liquibase.version}</version>
         </dependency>
 
         <dependency>

